src/query/catalog/src/catalog.rs

```diff
@@ -151,24 +151,40 @@ pub trait Catalog: DynClone + Send + Sync {
 
     async fn upsert_table_option(
         &self,
+        tenant: &str,
+        db_name: &str,
         req: UpsertTableOptionReq,
     ) -> Result<UpsertTableOptionReply>;
 
-    async fn update_table_meta(&self, req: UpdateTableMetaReq) -> Result<UpdateTableMetaReply>;
+    async fn update_table_meta(
+        &self,
+        tenant: &str,
+        db_name: &str,
+        req: UpdateTableMetaReq,
```

Comment:
If we already passed `tenant` and `db` args, how about passing `TableId` directly? So that FUSE engine can build `UpdateTableMetaReq` byself.

And catalog like `dive` doesn't need to know about `UpdateTableMetaReq` anymore.